### PR TITLE
Bug/use pool

### PR DESCRIPTION
**Pull request by crbrox**
A partial fix for #2622 

@fgalan

### CODE REVIEW

File: src/lib/ngsiNotify/QueueNotifier.cpp [hunk 1]
```diff
@@ -63,7 +63,7 @@ int QueueNotifier::start()
 void QueueNotifier::sendNotifyContextRequest
```
Comment (fgalan) [inline src/lib/ngsiNotify/QueueNotifier.cpp]: CNR entry missing. Suggested line:

> Fix: threadpool mode was not actually used, even if -notificationMode is set so; this fix solves the problem for not-custom notifications (partial fix for #2622)

File: src/lib/ngsiNotify/QueueNotifier.cpp [hunk 2]
```diff
@@ -75,6 +75,26 @@ void QueueNotifier::sendNotifyContextRequest
 {
   ConnectionInfo ci;
 
+
+  // Now, if it is a "custom" notification (with template)
+  // we delegate to parent method, and do not use the pool
+  // of threads. (To be changed in a future version)
```
Comment (fgalan) [inline src/lib/ngsiNotify/QueueNotifier.cpp]: For completeness, a reference to #2622 could be a good idea.

Comment (crbrox) [inline src/lib/ngsiNotify/QueueNotifier.cpp]: Fixed 48f6fc4

File: src/lib/ngsiNotify/QueueNotifier.cpp (hunk 1 above)
Comment (crbrox) [inline src/lib/ngsiNotify/QueueNotifier.cpp]: Fixed 1d399ab

**Comment by fgalan** [comment]
LGTM

**Pull request merged**